src/patternfly/components/Compass/compass.scss

```diff
@@ -92,6 +94,24 @@
   min-height: 0;
 }
 
+.#{$compass}__main-header {
+  & .#{$compass}__panel {
+    display: flex;
+    flex-wrap: wrap;
+    gap: var(--#{$compass}__main-header--RowGap) var(--#{$compass}__main-header--ColumnGap);
+    align-items: center;
+
+    .#{$compass}__main-header-title {
+      flex-grow: 1;
+      max-width: 100%;
```

Comment:
Curious why you needed `max-width: 100%`? Sometimes the contents of flex items overflow their container, but it's most often due to `min-width: min-content` as the default for a flex item, and you'd disable that with `min-width: 0;`.